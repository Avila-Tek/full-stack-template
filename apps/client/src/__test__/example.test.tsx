import React from 'react';
import { render } from '@testing-library/react';
import Page from '../app/page';

jest.mock('next/font/google', () => ({
  Inter: jest.fn(() => 'MockedInter'),
}));

it('renders homepage unchanged', () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
