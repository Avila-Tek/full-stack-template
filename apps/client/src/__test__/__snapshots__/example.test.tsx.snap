// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    Home
  </div>
</div>
`;
